packages/api/resolvers/mutations/setUserTags.js

```diff
@@ -11,14 +11,5 @@ export default function setUserTags(
   if (!normalizedUserId) throw new InvalidIdError({ normalizedUserId });
   const user = Users.findOne({ _id: normalizedUserId });
   if (!user) throw new UserNotFoundError({ normalizedUserId });
-  Users.update(
-    { _id: normalizedUserId },
-    {
-      $set: {
-        updated: new Date(),
-        tags,
-      },
-    }
-  );
-  return Users.findOne({ _id: normalizedUserId });
+  return user.addTags({ tags });
```

Comment:
addTags means that the already added tags are preserved, this should be setTags because it actually overwrites all tags with the new array.